src/blob_serializer_deserializer-inl.h

```diff
@@ -282,6 +286,14 @@ size_t BlobSerializer<Impl>::WriteString(const std::string& data) {
   return written_total;
 }
 
+template <typename Impl>
+size_t BlobSerializer<Impl>::WriteString(const std::string& data) {
```

Comment:
```suggestion
size_t BlobSerializer<Impl>::WriteString(const std::string_view data) {
```

Reply:
This is the method that writes strings (which logs the string directly in debug mode). The one that doesn't is `WriteStringView` (which obviously doesn't log that data as string).